src/test/java/com/github/kokorin/jaffree/ffmpeg/FFmpegTest.java

```diff
@@ -392,6 +392,21 @@ public void consume(Frame frame) {
         Assert.assertNotNull(result);
     }
 
+    @Test
+    public void testUserAgent() throws Exception{
+      FFmpegResult result;
+      String URL = "https://static.videezy.com/system/protected/files/000/007/213/Biking_Girl_Alpha.mov?md5=zJB3WS6tzcdWmKjzHnSTLA&expires=1553233302";
+
+      result = FFmpeg.atPath(BIN)
+              .addInput(UrlInput
+                      .fromUrl(URL)
+                      .setUserAgent("Jaffree/0.9.4")
+              )
+              .addOutput(new NullOutput())
+              .execute();
+
+      Assert.assertNotNull(result);
```

Comment:
Headers are not actually checked

Reply:
In the new test, the test will fail if FFmpeg detects a 4xx status code.